framework/source/class/qx/bom/AnimationFrame.js

```diff
@@ -76,11 +76,13 @@ qx.Bootstrap.define("qx.bom.AnimationFrame",
      * soon as the given duration is over.
      *
      * @param duration {Number} The duration the sequence should take.
+     *
+     * @ignore(performance.*)
      */
     startSequence : function(duration) {
       this.__canceled = false;
 
-      var start = +(new Date());
+      var start = (window.performance && window.performance.now) ? (performance.now() + qx.bom.AnimationFrame.__start) : Date.now();
```

Comment:
I'm not shure, but this should be tested against Internet Explorer which tends to fail on such tests like 
```javascript
(window.performance && window.performance.now)
```
vs.
```javascript
("performance" in window ...
```

I think it whould be better to have a ```qx.core.Envorinment.get("performance.timer")``` to check against, which also may be handy in other situations.